csharp-basic-topics/ValidateNumberIsPositiveOrNegative/ValidateNumberIsPositiveOrNegative/NumberValidation.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ValidateNumberIsPositiveOrNegative
+{
+    public static class NumberValidation
+    {
+        public static int IsPositiveOrNegativeUsingConditionalMethod(int number)
+        {
+            var result = 0;
+
+            if (number > 0)
+            {
+                result = 1;
+            }
+            else if (number < 0)
+            {
+                result = - 1;
+            }
+            else
+            {
+                result = 0;
+            }
+            return result;
+        }
```

Comment:
There must be one line of space between each method in a class

Reply:
Added space in the class.